lib/atlas/scaler.rb

```diff
@@ -9,16 +9,30 @@ def initialize(base_dataset_key, derived_dataset_name, number_of_residences)
     def create_scaled_dataset
       derived_dataset = Dataset::DerivedDataset.new(
         @base_dataset.attributes
-        .merge(scaled_attributes)
+        .merge(AreaAttributesScaler.call(@base_dataset, scaling_factor))
         .merge(new_attributes))
 
       derived_dataset.save!
 
-      GraphPersistor.call(@base_dataset, derived_dataset.graph_path)
+      GraphPersistor.call(@base_dataset, derived_dataset.graph_path, export_modifier: Scaler::GraphScaler.new(scaling_factor))
```

Comment:
The thing is that I would like to avoid the word "scaling" completely inside graph_persistor.rb since the latter has nothing to do with the former.